app/services/retrieve_pipeline_viz_graph_data_service.rb

```diff
@@ -241,7 +241,10 @@ def modify_step_name(step_name)
 
   def remove_host_filtering_urls(edges)
```

Comment:
could you add a comment why it is important here to remove host filtering urls? is it for privacy? 

and would those files then be a good candidate for `additional_output_files_hidden` ?

Reply:
Yes. This is because the data may contain PGI. 

No, I know this is confusing but those files are hidden in all contexts. These files are available to the pipeline runner, and their existence is visible to everyone. We probably could expose them but we wanted to err on the side of caution when it comes to making any files available from this stage. Will explain in a comment.